deployment/community/.env.template

```diff
@@ -29,12 +29,11 @@ TEMP_DIR=/data/tmp
 
 #DB_APPLICATION_NAME=mergin
 
-#DB_DATABASE=postgres
+DB_DATABASE=mergin
```

Comment:
why to change this? I think for production they need to change also user and password, create db, grant permissions, ...

so either we keep dummy (not safe) defaults or have a bootstrap script to make it properly